pipeline-templates/job/codebuild.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  TargetProjects: 'src/**/*.csproj'
```

Comment:
Can probably bin this variable if we're got the below code hardcoded.

Reply:
if they are all hard coded, you can remove that variable, but at the same  time, why we are not using that paramete?